oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGCSupport.java

```diff
@@ -98,12 +98,15 @@ public Iterable<NodeDocument> getPossiblyDeletedDocs(final long fromModified, fi
      * @return matching documents.
      */
     public Iterable<NodeDocument> getModifiedDocs(final long fromModified, final long toModified, final int limit,
-                                                  @NotNull final String fromId) {
+                                                  @NotNull final String fromId, final Set<String> includePaths,
```

Comment:
I would suggest to make `includePaths` & `excludePaths` as `@NotNull`. It would avoid us in all the null checks in the later code.

Reply:
done in https://github.com/apache/jackrabbit-oak/pull/1547/commits/14ba933fad9b8dadcd51d00aa2e186a0b3e39568